app/src/main/java/com/example/jwstimager/IconData.kt

```diff
@@ -4,33 +4,33 @@ object IconData {
     val IconDataList = listOf<ImageData>(
```

Comment:
What was IconData.kt used for? I had to comment some stuff out when I changed some of the attributes of the ImageData class. Do we still need this file?

Reply:
IconData.kt is a depreciated data structure for nav. It should have no dependencies and can be deleted.